src/algebra/lie/basic.lean

```diff
@@ -181,6 +181,9 @@ instance : has_coe (L₁ →ₗ⁅R⁆ L₂) (L₁ →ₗ[R] L₂) := ⟨morphis
 /-- see Note [function coercion] -/
 instance : has_coe_to_fun (L₁ →ₗ⁅R⁆ L₂) := ⟨_, morphism.to_fun⟩
 
+@[simp] lemma coe_mk (f : L₁ → L₂) (h₁ h₂ h₃) :
+  ((morphism.mk f h₁ h₂ h₃ : L₁ →ₗ⁅R⁆ L₂) : L₁ → L₂) = f := rfl
```

Comment:
```suggestion
  ((⟨f, h₁, h₂, h₃⟩ : L₁ →ₗ⁅R⁆ L₂) : L₁ → L₂) = f := rfl
```
I slightly prefer this but feel free to ignore!

Reply:
`monoid_hom.coe_mk` does it my way, `ring_hom.coe_mk` does it your way. So I can't really make a consistency argument either way.